<ui version="4.0" >
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>JACK Audio Connection Kit - Qt GUI Interface.

   Copyright (C) 2003-2014, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qjackctlSocketForm</class>
 <widget class="QDialog" name="qjackctlSocketForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Socket - JACK Audio Connection Kit</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="qjackctl.qrc" >:/images/patchbay1.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="SocketTabWidget" >
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;Socket</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>4</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="SocketNameTextLabel" >
         <property name="text" >
          <string>&amp;Name (alias):</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>SocketNameLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2" >
        <widget class="QLineEdit" name="SocketNameLineEdit" >
         <property name="toolTip" >
          <string>Socket name (an alias for client name)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2" >
        <widget class="QComboBox" name="ClientNameComboBox" >
         <property name="toolTip" >
          <string>Client name (regular expression)</string>
         </property>
         <property name="editable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2" >
        <widget class="QPushButton" name="PlugAddPushButton" >
         <property name="toolTip" >
          <string>Add plug to socket plug list</string>
         </property>
         <property name="text" >
          <string>Add P&amp;lug</string>
         </property>
         <property name="icon" >
          <iconset resource="qjackctl.qrc" >:/images/add1.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="PlugNameTextLabel" >
         <property name="text" >
          <string>&amp;Plug:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>PlugNameComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QComboBox" name="PlugNameComboBox" >
         <property name="toolTip" >
          <string>Port name (regular expression)</string>
         </property>
         <property name="editable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item rowspan="5" row="4" column="1" >
        <widget class="QTreeWidget" name="PlugListView" >
         <property name="contextMenuPolicy" >
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="toolTip" >
          <string>Socket plug list</string>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights" >
          <bool>true</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Socket Plugs / Ports</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="4" column="2" >
        <widget class="QPushButton" name="PlugEditPushButton" >
         <property name="toolTip" >
          <string>Edit currently selected plug</string>
         </property>
         <property name="text" >
          <string>&amp;Edit</string>
         </property>
         <property name="icon" >
          <iconset resource="qjackctl.qrc" >:/images/edit1.png</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="2" >
        <widget class="QPushButton" name="PlugRemovePushButton" >
         <property name="toolTip" >
          <string>Remove currently selected plug from socket plug list</string>
         </property>
         <property name="text" >
          <string>&amp;Remove</string>
         </property>
         <property name="icon" >
          <iconset resource="qjackctl.qrc" >:/images/remove1.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="ClientNameTextLabel" >
         <property name="text" >
          <string>&amp;Client:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>ClientNameComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="2" >
        <widget class="QPushButton" name="PlugDownPushButton" >
         <property name="toolTip" >
          <string>Move down currently selected plug in socket plug list</string>
         </property>
         <property name="text" >
          <string>&amp;Down</string>
         </property>
         <property name="icon" >
          <iconset resource="qjackctl.qrc" >:/images/down1.png</iconset>
         </property>
        </widget>
       </item>
       <item row="7" column="2" >
        <widget class="QPushButton" name="PlugUpPushButton" >
         <property name="toolTip" >
          <string>Move up current selected plug in socket plug list</string>
         </property>
         <property name="text" >
          <string>&amp;Up</string>
         </property>
         <property name="icon" >
          <iconset resource="qjackctl.qrc" >:/images/up1.png</iconset>
         </property>
        </widget>
       </item>
       <item row="6" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>8</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="1" >
        <widget class="QCheckBox" name="ExclusiveCheckBox" >
         <property name="toolTip" >
          <string>Enforce only one exclusive cable connection</string>
         </property>
         <property name="text" >
          <string>E&amp;xclusive</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" >
        <widget class="QLabel" name="SocketForwardTextLabel" >
         <property name="text" >
          <string>&amp;Forward:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>SocketForwardComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="1" colspan="2" >
        <widget class="QComboBox" name="SocketForwardComboBox" >
         <property name="toolTip" >
          <string>Forward (clone) all connections from this socket</string>
         </property>
         <property name="editable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2" >
        <widget class="QGroupBox" name="SocketTypeGroupBox" >
         <property name="title" >
          <string>Type</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>4</number>
          </property>
          <item>
           <widget class="QRadioButton" name="AudioRadioButton" >
            <property name="toolTip" >
             <string>Audio socket type (JACK)</string>
            </property>
            <property name="text" >
             <string>&amp;Audio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="MidiRadioButton" >
            <property name="toolTip" >
             <string>MIDI socket type (JACK)</string>
            </property>
            <property name="text" >
             <string>&amp;MIDI</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="AlsaRadioButton" >
            <property name="toolTip" >
             <string>MIDI socket type (ALSA)</string>
            </property>
            <property name="text" >
             <string>AL&amp;SA</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="DialogButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="8" />
 <tabstops>
  <tabstop>SocketTabWidget</tabstop>
  <tabstop>SocketNameLineEdit</tabstop>
  <tabstop>AudioRadioButton</tabstop>
  <tabstop>MidiRadioButton</tabstop>
  <tabstop>AlsaRadioButton</tabstop>
  <tabstop>ClientNameComboBox</tabstop>
  <tabstop>PlugNameComboBox</tabstop>
  <tabstop>PlugAddPushButton</tabstop>
  <tabstop>PlugListView</tabstop>
  <tabstop>PlugEditPushButton</tabstop>
  <tabstop>PlugRemovePushButton</tabstop>
  <tabstop>PlugUpPushButton</tabstop>
  <tabstop>PlugDownPushButton</tabstop>
  <tabstop>ExclusiveCheckBox</tabstop>
  <tabstop>SocketForwardComboBox</tabstop>
  <tabstop>DialogButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="qjackctl.qrc" />
 </resources>
 <connections/>
</ui>
